src/OSPSuite.SimModelNative/src/Species.cpp

```diff
@@ -164,15 +164,12 @@ bool Species::IsConstant(bool forCurrentRunOnly)
 
 double Species::GetValue (const double * y, double time, ScaleFactorUsageMode scaleFactorMode)
 {
-//	assert(_valueFormula==NULL);
-
-	if (!IsConstantDuringCalculation())
+	if ((time > _simulationStartTime) && !IsConstantDuringCalculation())
 	{
-		assert(time == _simulationStartTime);
+		return(y[m_ODEIndex]);
 	}
```

Comment:
Debug asserts are really evil: in the Debug mode I saw immediately that it crashes here.
we should replace all of them with throw exception.